src/main/java/org/sopt/confeti/api/performance/controller/PerformanceController.java

```diff
@@ -61,10 +61,9 @@ public ResponseEntity<BaseResponse<?>> getPerformReservationInfo(
     @GetMapping("/association/{artistId}")
     public ResponseEntity<BaseResponse<?>> getPerformanceByArtist(
             @UserId(require = false) Long userId,
-            @PathVariable(name="artistId") String artistId,
-            @RequestParam(name="cursor", required = false) Long cursor
+            @PathVariable(name="artistId") String artistId
     ){
-        PerformanceByArtistDTO performances = performanceFacade.getPerformanceByArtistId(userId, artistId, cursor);
+        PerformanceByArtistDTO performances = performanceFacade.getPerformanceByArtistId(userId, artistId);
```

Comment:
P3. 여러 개의 공연을 반환해야하기 때문에 이렇게 수정되어야할 것 같아요! (네이밍)

```suggestion
        PerformancesByArtistDTO performances = performanceFacade.getPerformancesByArtistId(userId, artistId);
```

Reply:
아래와 같이 복수형으로 네이밍 수정해 두었습니다! 아티스트 연관 공연 조회의 의미를 나타내기 위해서는 기존의 `PerformancesByArtist` 보다 `ArtistPerformances`가 더 적절할 것 같아 이 부분도 수정해 두었습니다! 
- `PerformancesByArtistDTO` -> `ArtistPerformancesDTO`
- `getPerformanceByArtistId` -> `getPerformancesByArtistId`

Reply:
네이밍의 천재 네이밍의 고수